apps/api/src/modules/learner/controllers/auth.learner_quiz.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Get, Inject, Param, ParseIntPipe } from '@nestjs/common';
+import { TYPES } from '../interfaces';
+import { AuthController } from 'src/utils/decorators/auth-controller.decorator';
+import { Roles } from 'src/modules/auth/decorators/roles.decorators';
+import { Role } from 'src/modules/user/domain/role.enum';
+import { SpaceId } from 'src/modules/auth/decorators';
+import { IGetLearnerService } from '../interfaces/services/get.learner.service.interface';
+import { ApiLogger } from '../logger/api-logger.service';
+import { TYPES as QUIZ_TYPES} from '../../quiz/interfaces'
+import { IValidateSpaceQuizService } from 'src/modules/quiz/interfaces/services/validate-space.quiz.service.interface';
+import { IGetAssignedLearnerService } from '../interfaces/services/get-assigned.learner.service.interface';
+@AuthController('learner-quiz')
+export class AuthLearnerQuizController {
```

Comment:
delete unused import & add enter between imports and code

```suggestion
import { Get, Inject, Param, ParseIntPipe } from '@nestjs/common';
import { TYPES } from '../interfaces';
import { AuthController } from 'src/utils/decorators/auth-controller.decorator';
import { Roles } from 'src/modules/auth/decorators/roles.decorators';
import { Role } from 'src/modules/user/domain/role.enum';
import { SpaceId } from 'src/modules/auth/decorators';
import { ApiLogger } from '../logger/api-logger.service';
import { TYPES as QUIZ_TYPES} from '../../quiz/interfaces'
import { IValidateSpaceQuizService } from 'src/modules/quiz/interfaces/services/validate-space.quiz.service.interface';
import { IGetAssignedLearnerService } from '../interfaces/services/get-assigned.learner.service.interface';

@AuthController('learner-quiz')
```

Reply:
haha sorry